src/java/org/apache/cassandra/tools/nodetool/SetStreamThroughput.java

```diff
@@ -23,11 +23,11 @@
 import org.apache.cassandra.tools.NodeProbe;
 import org.apache.cassandra.tools.NodeTool.NodeToolCmd;
 
-@Command(name = "setstreamthroughput", description = "Set the Mb/s throughput cap for streaming and entire SSTable streaming in the system, or 0 to disable throttling")
+@Command(name = "setstreamthroughput", description = "Set the megabits per second throughput cap for streaming and entire SSTable streaming in the system, or 0 to disable throttling")
 public class SetStreamThroughput extends NodeToolCmd
 {
     @SuppressWarnings("UnusedDeclaration")
-    @Arguments(title = "stream_throughput", usage = "<value_in_mb>", description = "Value in Mb, 0 to disable throttling", required = true)
+    @Arguments(title = "stream_throughput", usage = "<value_in_mb>", description = "Value in megabits, 0 to disable throttling", required = true)
```

Comment:
nit: Should everything here be `MiB/s`?

Reply:
Nope, this is a bit tricky...
the old one property is still set in megabits as the property is pre-4.0. No breaking change
The other one is new and was changed to MiB/s.... it is a bit weird, but we didn't want breaking changes....new properties should be also with the new units...

Reply:
`setEntireSSTableStreamThroughput()` expects MiB though, right? The value of `stream_throughput` must be specified in MiB if we use `--entire-sstable-throughput`?

Reply:
i.e. Should we just say "Set the throughput cap..." in the description and have the `description` indicate the value depends on whether `--entire-sstable-throughput` is set?

Reply:
`The value of stream_throughput must be specified in MiB if we use --entire-sstable-throughput`
Correct, otherwise megabits if it is not used because it is pre-4.0 :( 
I corrected the `description` as per your recommendation